crates/nu-command/Cargo.toml

```diff
@@ -14,6 +14,7 @@ bench = false
 
 [dependencies]
 nu-ansi-term = "0.46.0"
+nu-cli = { path = "../nu-cli", version = "0.76.1" }
```

Comment:
Mhh that introduces a circular dependency situation as `nu-cli` itself depends on `nu-command`.

Reply:
Hmm I'm just using `eval_hook` from there. Where do you think this should be moved?

Reply:
I moved this in a new commit.